src/js/engagement_view/src/modules/GraphViz/graph/mergeGraphs.tsx

```diff
@@ -43,6 +44,7 @@ export const mergeGraphs = (curGraph: MergeGraphType, update: MergeGraphType): M
             }
         } else {
             nodes.set(newNode.uid, newNode);
+            console.log('new node added ', newNode);
```

Comment:
We should make this into a user message at some point, not for this PR though

Reply:
Yeah, that's a good idea. It would be cool to display some info like that.